docs/source/analysis.ipynb

```diff
@@ -0,0 +1,530 @@
+    "        ├── grid_movies/\n",
+    "        ├── trajectory_plots/\n",
+    "        ├── checkpoint.p\n",
+    "        └── results.h5\n",
+    "    ...\n",
+    "    └── progress.yaml\n",
+    "\n",
+    "```"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Set up notebook for interactive widgets\n",
+    "The analysis notebook uses interactive widgets to help you visualize and explore the data. To enable the widgets, the first time you run this notebook, you need to run the cell below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Install and Enable widget extensions configurator\n",
+    "!jupyter nbextension install --py jupyter_nbextensions_configurator --sys-prefix\n",
+    "!jupyter nbextension enable --py --sys-prefix widgetsnbextension\n",
+    "!jupyter nbextension enable --py --sys-prefix  --py qgrid\n",
+    "\n",
+    "# Install bokeh extensions\n",
+    "!jupyter nbextension install --sys-prefix --symlink --py jupyter_bokeh\n",
+    "!jupyter nbextension enable jupyter_bokeh --py --sys-prefix"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load Progress\n",
+    "\n",
+    "**[IMPORTANT] ALWAYS run this cell when you open this notebook.** If you don't, the analysis functions won't work. Please ensure you run the Load Progress cell below before running any analysis modules.\n",
+    "\n",
+    "MoSeq uses a [progress.yaml](https://github.com/dattalab/moseq2-app/wiki/Directory-Structures-and-yaml-Files-in-MoSeq-Pipeline#progressyaml) file to keep track of the progress. \n",
+    "\n",
+    "The following cell generates a `progress.yaml` file or loads your progress from the last saved checkpoint.  This cell checks if there is a  `progress.yaml` in the project folder and generate/load a dictionary contains the necessary paths to use the features in the rest of the notebook. The cell output displays the content in `progress_paths`.\n",
+    "\n",
+    "- **Specify the filename to the progress file**. No change is needed if the notebooks are in the project folder.\n",
+    "- **Run the following cell**."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from keypoint_moseq.analysis import track_progress\n",
+    "\n",
+    "project_dir = './' # project directory\n",
+    "model_dirname = 'model_dir' # model directory name for the model to analyze\n",
+    "input_dir = './videos' # input video directory path\n",
+    "progress_filename = 'progress.yaml' # progress file name\n",
+    "\n",
+    "progress_paths = track_progress(model_dirname, project_dir, input_dir, 'progress.yaml')"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Select a different model to analyze"
```

Comment:
@calebweinreb What's the point of this? shouldn't they just put the model they want to analze once at the top of the notebook and then be done?

That's to make keypoint moseq steps similar to that of depth moseq, i removed it here: 7fc41964592a9aa3d633d44692680ab3f8a80b62